### ISSUE TITLE
Countdown detection should also work with ROOMDATA messages

### ISSUE THREAD

**Issue by ZombieAlex** [ZombieAlex/PushMFC#1]
The existing countdown detection logic needs to stay as-is, because many models don't actually use the automatic countdowns that MFC added.  They do it manually.  But when we detect via ROOMDATA messages from the server that a model _is_ using an automatic countdown, then we don't need any heuristics and we can immediately notify that a countdown is detected and exactly which number represents the current goal.

Of course the existing heuristic logic does eventually detect even automatic countdowns, usually. But it's guessing and delayed. Using ROOMDATA would be more accurate and immediate.

**Comment by ZombieAlex** [comment]
Of course the existing heuristic logic does eventually detect even automatic countdowns, usually.  But it's guessing and delayed.  Using ROOMDATA would be more accurate and immediate.

**Comment by ZombieAlex** [comment]
The major issue with this idea is that I don't believe we receive ROOMDATA packets for rooms we're not currently in.  So this couldn't work as well for monitoring all of MFC.  That's why this is a low priority and likely not something I'll ever get to.

Further, if I implement this, it will be part of the mfcauto-plugins repo, where I have moved the Countdown class too.

Closing this work item in this repo.  If I ever make the change in the plugin, PushMFC will get it automatically.
